frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/coloring/PrintElementColorManager.java

```diff
@@ -0,0 +1,29 @@
+package com.virtuslab.gitmachete.frontend.graph.coloring;
+
+import lombok.AllArgsConstructor;
+
+import com.virtuslab.gitmachete.frontend.graph.api.GraphEdge;
+import com.virtuslab.gitmachete.frontend.graph.api.GraphNode;
+import com.virtuslab.gitmachete.frontend.graph.api.IGraphElement;
+import com.virtuslab.gitmachete.frontend.graph.items.IGraphItem;
+import com.virtuslab.gitmachete.frontend.graph.repository.RepositoryGraph;
+
+@AllArgsConstructor
+public class PrintElementColorManager {
+  private final RepositoryGraph repositoryGraph;
+
+  public int getColorId(IGraphElement element) {
+    int nodeIndex;
+    if (element instanceof GraphNode) {
```

Comment:
Maybe add `isNode`, `asNode`, `asEdge` to `IGraphElement` (see `com.virtuslab.gitmachete.backend.api.BaseGitMacheteBranch`)